pkg/common-controller/framework_test.go

```diff
@@ -770,6 +761,7 @@ func newTestController(kubeClient kubernetes.Interface, clientset clientset.Inte
 	ctrl.snapshotListerSynced = alwaysReady
 	ctrl.classListerSynced = alwaysReady
 	ctrl.pvcListerSynced = alwaysReady
+	ctrl.createSnapshotContentInterval = time.Millisecond * 10
```

Comment:
Set to 5 * time.Millisecond.

Also add a ctrl.createSnapshotContentRetryCount = 3